docs/conf/conf.py

```
@@ -99,5 +104,10 @@
 # https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/static_assets.html
 
 
-def setup(app):
+def setup(app) -> None:  # noqa: ANN001 Ignore type annotation for Sphinx setup function.
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `ANN001`)

Remove unused `noqa` directive

(RUF100)